coredumper/coredumper.py

```diff
@@ -0,0 +1,407 @@
+#!/usr/bin/env python3
+
+from __future__  import print_function
+import argparse
+import struct
+import shutil
+import os
+import sys
+import itertools
+from ctypes      import LittleEndianStructure, c_uint32, c_char
+from collections import namedtuple
+from operator    import attrgetter
+from functools   import partial
+
+ArchDefinition = namedtuple('ArchDefinition', ['name', 'bitness', 'endianness'])
+VALID_ARCHS = {}
```

Comment:
Is this host architecture or DSP architecture ?

Reply:
This is architecture of the coredump file, so I assume it is DSP architecture.

Reply:
ok, I'm confused as it's checking for x86 or amd64, but this looks like this is more to do with data width (i.e. size of pointer/int) on the host. If so, can we rename this option for host data width as it make it more obvious the script is not just for running on x86 or amd64 architectures ?

Reply:
Changed to LE32bit and LE64bit to avoid confusion.
Also left as a comment BE32bit and BE64bit - I don't know if we'll ever need them, but is it only 6 lines of commentary.